src/aws/osml/model_runner/app_config.py

```diff
@@ -77,9 +77,9 @@ class BotoConfig:
     sagemaker: (Config) the sagemaker specific boto client configuration
     """
 
-    # required env configuration
     default: Config = Config(region_name=ServiceConfig.aws_region, retries={"max_attempts": 15, "mode": "standard"})
     sagemaker: Config = Config(region_name=ServiceConfig.aws_region, retries={"max_attempts": 30, "mode": "adaptive"})
+    ddb: Config = Config(region_name=ServiceConfig.aws_region, retries={"max_attempts": 100, "mode": "adaptive"})
```

Comment:
This seems like a big number given that the default value is 5 and the batch_write has a secondary retry loop that defaults to 40. Seems like we could retry a batch 4000 times which seems excessive. 

Can we get some metrics on the actual number of retries used during the load tests and validate that this is necessary?

Reply:
I actually had reduced this to 50 locally and found it working for the scenario and that never made it back up into this commit. I will try and drive into what the most logical number would be.